src/tile/FeaturedTile.js

```diff
@@ -116,14 +116,18 @@ const FeaturedTile = props => {
           >
             {title}
           </Text>
-          <Text
-            style={StyleSheet.flatten([
-              styles.text,
-              captionStyle && captionStyle,
-            ])}
-          >
-            {caption}
-          </Text>
+          {typeof caption === 'string' ? (
```

Comment:
Is it possible that you can use the approach here https://github.com/react-native-training/react-native-elements/blob/master/src/list/ListItem.js#L37

We have a helper for doing this kind of thing.